run_electrum

```diff
@@ -365,12 +365,6 @@ def main():
     if not config_options.get('verbosity'):
         warnings.simplefilter('ignore', DeprecationWarning)
 
-    # check if we received a valid payment identifier
-    uri = config_options.get('url')
-    if uri and not PaymentIdentifier(None, uri).is_valid():
-        print_stderr('unknown command:', uri)
-        sys.exit(1)
-
```

Comment:
I see, this explains why it didn't work for testnet.

Reply:
Yes, this confusion caused me to doubt my cmdline fu :)

Might be better to move the passing of payment identifiers via a switched param (e.g. --uri or something similar) in order to disambiguate from a command

Reply:
last time I checked, it was not possible to hook it to firefox with a --uri parameter

Reply:
<a href="bitcoin:donate.getmonero.org">this is a test</a>

Reply:
maybe we could provide a separate script for firefox, that launches `run_electrum --uri $1`

Reply:
On linux we can use the `.desktop` file spec which allows us to sculpt the cmdline and let firefox delegate to `xdg-open`.

Reply:
I see. I was referring to firefox's `network.protocol-handlers` in `about:config`, which does not seem to allow that
I'll open a separate issue about it.